pandas/core/common.py

```diff
@@ -2646,7 +2646,7 @@ def _astype_nansafe(arr, dtype, copy=True):
     return arr.view(dtype)
 
 
-def _clean_fill_method(method):
+def _clean_fill_method(method, allow_nearest=False):
```

Comment:
yes, when nearest is valid for fillna we should remove this method.

I was considering adding nearest for fillna in this PR but I'd rather save it for another just to minimize the scope here.